README.md

```diff
@@ -1,118 +1,145 @@
-# o-ft-icons [![Build Status](https://circleci.com/gh/Financial-Times/o-ft-icons.png?style=shield&circle-token=bd79f68bf3b7af081ef39c62e493e737d56d2958)](https://circleci.com/gh/Financial-Times/o-ft-icons)
+# o-icons [![Build Status](https://circleci.com/gh/Financial-Times/o-icons.png?style=shield&circle-token=bd79f68bf3b7af081ef39c62e493e737d56d2958)](https://circleci.com/gh/Financial-Times/o-icons)
 
 Icon font with helper classes, and resolution-independent SVG icons to load via the [responsive image service](http://image.webservices.ft.com).
 
 ## Quick start
 
 ```html
 <!-- Load web fonts with @font-face declarations  -->
-<link rel="stylesheet" href="//build.origami.ft.com/v2/bundles/css?modules=o-ft-icons@^2.0.0" />
+<link rel="stylesheet" href="//origami-build.ft.com/v2/bundles/css?modules=o-icons@^2.0.0" />
 
 <!-- In your markup, use the helper classes, such as: -->
-<i class="o-ft-icons-icon o-ft-icons-icon--arrow-down"></i>
+<i class="o-icons-icon o-icons-icon--arrow-down"></i>
 ```
 
-[Complete list of available icons](http://build.origami.ft.com/v2/files/o-ft-icons@latest/demos/main.html)
+[Complete list of available icons](http://origami-build.ft.com/v2/files/o-icons@latest/demos/all-icons.html)
 
 ## Advanced usage
 
 There are multiple ways to use the icons:
 
 1. Using the CSS helper classes
-1. Extending the predefined Sass placeholders into your own CSS classes
-1. Resolution independent SVGs, using the [responsive image service](http://image.webservices.ft.com/)
+1. Including the predefined Sass mixins into your own CSS classes
+1. Manually using the [responsive image service](https://image.webservices.ft.com)
 
 ### 1. Using the CSS helper classes
 
 ```scss
 // public/bundle.scss
 
-$o-ft-icons-is-silent: false;
-@import "o-ft-icons/main";
+$o-icons-is-silent: false;
+@import "o-icons/main";
 ```
 
 ```html
 <!-- In your markup, use the helper classes, such as: -->
-<i class="o-ft-icons-icon o-ft-icons-icon--columnists"><i>
+<i class="o-icons-icon o-icons-icon--plus"><i>
 ```
 
-### 2. Extending the predefined Sass placeholders into your own CSS classes
+When using CSS classes, it isn't possible to set a colour for the icon or to specify a size for the PNG fallback. The defaults will be 'black' for the icon colour and '128px' for the icon size.
```

Comment:
The defaults are `black` for the icon colour and `128px` for the width and height.